app/views/tracks/approaches/show.html.haml

```diff
@@ -1,17 +1,23 @@
 #page-approaches-show.pb-80
   %header.pt-24.md:pt-48.pb-64.md:pb-128
     .md-container.flex.flex-col.md:flex-row-reverse.items-center.c-shapes.c-shapes-1{ class: 'gap-[48px]' }
-      .py-8.px-24.bg-gray.border-2.border-gradient.rounded-100.flex.flex-row.items-center{ class: "h-[68px]" }
-        = graphical_icon "balloons", category: :graphics, css_class: "h-[40px] mr-12"
-        = graphical_icon "building", category: :graphics, css_class: "h-48 mr-12"
-        -# = track_icon css_class: "h-[40px] w-[40px] mr-12"
-          . = exercise_icon "amusement-park", css_class: "h-48 mr-12"
-        .flex.flex-col
-          .text-h5.whitespace-nowrap Amusement park
-          .textColor-6.font-normal.leading-150.text-16 Csharp
+      = link_to Exercism::Routes.track_exercise_path(@track, @exercise) do
+        .py-8.px-24.bg-gray.border-2.border-gradient-bgGray.rounded-100.flex.flex-row.items-center{ class: "h-[68px]" }
```

Comment:
Merge these two lines. If you ever have a single div within a link, then normally the div's classes can be inlined into the link.

Reply:
true, sorry!